lib/dnsimple/client/clients.rb

```diff
@@ -43,7 +43,14 @@ def webhooks
 
 
     # @!class Struct
-    class ClientService < ::Struct.new(:client)
+    class ClientService
```

Comment:
No, some changes require manual intervention. Luckly, most of them are handled automatically.